draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -89,7 +89,7 @@ DefaultStory.argTypes = {
     options: ["spot", "scene"],
     mapping: {
       spot: <Informative alt="" />,
-      scene: <HumanityAtWork alt="" />,
+      scene: <BrandMomentPositiveOutro alt="" />,
```

Comment:
If `BrandMomentPositiveOutro` is a replacement for `HumanityAtWork`, should we leave a deprecation notice for those who've already used `HumanityAtWork`?